app/containers/message/index.js

```diff
@@ -69,14 +70,29 @@ class MessageContainer extends React.Component {
 		theme: 'light'
 	}
 
-	componentDidMount() {
+	state = {}
+
+	async componentDidMount() {
 		const { item } = this.props;
 		if (item && item.observe) {
 			const observable = item.observe();
 			this.subscription = observable.subscribe(() => {
 				this.forceUpdate();
 			});
 		}
+
+		const db = database.active;
+		const usersCollection = db.collections.get('users');
+		try {
+			const user = await usersCollection.find(item.u._id);
```

Comment:
I guess we can do a relationship between messages and users to don't need this find, but I don't know if it's better for us right now.
WDYT?

Reply:
Let's stick with this code as it doesn't hurt our users experience.